static/app/views/insights/sessions/charts/crashFreeSessionsChartWidget.tsx

```diff
@@ -2,13 +2,16 @@ import ExternalLink from 'sentry/components/links/externalLink';
 import {tct} from 'sentry/locale';
 import {formatSeriesName} from 'sentry/views/dashboards/widgets/timeSeriesWidget/formatters/formatSeriesName';
 import {InsightsLineChartWidget} from 'sentry/views/insights/common/components/insightsLineChartWidget';
+import type {LoadableChartWidgetProps} from 'sentry/views/insights/common/components/widgets/types';
 import ChartSelectionTitle from 'sentry/views/insights/sessions/components/chartSelectionTitle';
 import useCrashFreeSessions from 'sentry/views/insights/sessions/queries/useCrashFreeSessions';
 import {CHART_TITLES} from 'sentry/views/insights/sessions/settings';
 import {SESSION_HEALTH_CHART_HEIGHT} from 'sentry/views/insights/sessions/utils/sessions';
 
-export default function CrashFreeSessionsChart() {
-  const {series, releases, isPending, error} = useCrashFreeSessions();
+export default function CrashFreeSessionsChart(props: LoadableChartWidgetProps) {
```

Comment:
This doesn't really matter bc they are default exports but to be consistent

```suggestion
export default function CrashFreeSessionsChartWidget(props: LoadableChartWidgetProps) {
```

Reply:
i forgot about these! thanks